src/modules/ContentCard.js

```diff
@@ -52,6 +52,18 @@ export default class ContentCard extends Component {
     return children;
   }
 
+  _renderWatchIcon (size) {
```

Comment:
Probably would be better to extract this into another file to be reusable since other components will be using this icon as well